client/ayon_core/tools/publisher/widgets/widgets.py

```diff
@@ -1615,6 +1636,21 @@ def _input_value_changed(self, value, attr_id):
         if not instances or not attr_def or not plugin_name:
             return
 
+        # Update the styling of the associated label if is default or override
+        label_widget = self._attr_def_id_to_label.get(attr_id)
+        if label_widget:
+            # NOTE: The default returned by attr_def.default is returning a
```

Comment:
I don't think this is true statement. If it this, it should not be handled here. Also comparing `float` and `int` should work ok.
```python
 1 == 1.0
 >>> True
 ```

Reply:
thanks. yep 1 == 1.0. 
i'm not sure what went on with my earlier testing. I was getting some difference in comparison.